src/components/fields/MultiTagArrayField.js

```diff
@@ -0,0 +1,163 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import BaseComponent from "../BaseComponent";
+import {classNames, getUiOptions} from "../../utils";
+import ReactContext from "../../ReactContext";
+import {Affix, Button} from "../components";
+
+
+@BaseComponent
+export default class MultiTagArrayField extends React.Component {
+	static contextType = ReactContext;
+	static propTypes = {
+		uiSchema: PropTypes.shape({
+			"ui:options": PropTypes.shape({
+				buttons: PropTypes.arrayOf(PropTypes.oneOfType(
+					[
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["moveAll"]),
+							fromField: PropTypes.string.isRequired,
+							toField: PropTypes.string.isRequired,
+							label: PropTypes.string,
+							className: PropTypes.string
+						}),
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["move"]),
+							toField: PropTypes.string.isRequired,
+							label: PropTypes.string,
+							className: PropTypes.string
+						}),
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["delete"]),
+							label: PropTypes.string,
+							className: PropTypes.string
+						})
+					]
+				))
+			})
+		}),
+		schema: PropTypes.shape({
+			type: PropTypes.oneOf(["object"])
+		}).isRequired,
+		formData: PropTypes.object
+	}
+
+	constructor(props) {
+		super(props);
+		this.state = {activeButtonIdx: undefined};
+	}
+
+	setContainerRef = (elem) => {
+		this.containerElem = elem;
+	}
+
+	getContainerRef = () => {
+		return this.containerElem;
+	}
+
+	render() {
+		const SchemaField = this.props.registry.fields.SchemaField;
+		const {Row, Col} = this.context.theme;
+		const {schema, uiSchema, idSchema, errorSchema, formData = {}} = this.props;
+		const uiOptions = getUiOptions(uiSchema);
+		const {buttons = []} = uiOptions;
+
+		const {activeButtonIdx} = this.state;
+
+		const propertyKeys = Object.keys(schema.properties)
+
+		return (
+			<Row className={`laji-form-multi-tag-array-field${activeButtonIdx !== undefined ? " laji-form-multi-tag-array-field-active" : ""}`}>
```

Comment:
There's a `classNames` utility from `/src/utils.tsx` that helps with constructing classes.

```
classNames("laji-form-multi-tag-array-field", activeButtonIdx && "laji-form-multi-tag-array-field-active")
```